app/views/shared/svg/_icon_sprite.html.erb

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+  <symbol id="svg-icon--plus" viewBox="0 0 22.354 22.354">
+    <polygon points="8.61,22.354 13.744,22.354 13.744,13.744 22.354,13.744 22.354,8.611 13.744,8.611 13.744,0 8.61,0 8.61,8.611 0,8.611 0,13.744 8.61,13.744" />
+  </symbol>
+  <symbol id="svg-icon--minus" viewBox="0 0 22.354 22.354">
+    <rect y="8.61" width="22.354" height="5.133" />
+  </symbol>
+  <symbol id="svg-icon--arrow-right" viewBox="0 0 22 34">
+    <polygon points="0,5 5,0 22,17 5.1,34 0,29 12,17" />
+  </symbol>
+  <symbol id="icon-arrow-right" viewBox="0 0 22 34">
```

Comment:
Yeah - scared of inadvertently breaking something as "icon-arrow-right" is being referenced elsewhere.